src/agent/border_agent.hpp

```diff
@@ -128,24 +128,28 @@ class BorderAgent
 
     void SetNetworkName(const char *aNetworkName);
     void SetExtPanId(const uint8_t *aExtPanId);
+    void SetThreadVersion(uint16_t aThreadVersion);
     void HandleThreadState(bool aStarted);
     void HandlePSKc(const uint8_t *aPSKc);
 
-    static void HandlePSKc(void *aContext, int aEvent, va_list aArguments);
-    static void HandleThreadState(void *aContext, int aEvent, va_list aArguments);
-    static void HandleNetworkName(void *aContext, int aEvent, va_list aArguments);
-    static void HandleExtPanId(void *aContext, int aEvent, va_list aArguments);
+    static void        HandlePSKc(void *aContext, int aEvent, va_list aArguments);
+    static void        HandleThreadState(void *aContext, int aEvent, va_list aArguments);
+    static void        HandleNetworkName(void *aContext, int aEvent, va_list aArguments);
+    static void        HandleExtPanId(void *aContext, int aEvent, va_list aArguments);
+    static void        HandleThreadVersion(void *aContext, int aEvent, va_list aArguments);
+    static const char *ThreadVersionToString(uint16_t aThreadVersion);
 
     Mdns::Publisher *mPublisher;
     Ncp::Controller *mNcp;
 
 #if OTBR_ENABLE_NCP_WPANTUND
     int mSocket;
 #endif
-    uint8_t mExtPanId[kSizeExtPanId];
-    char    mNetworkName[kSizeNetworkName + 1];
-    bool    mThreadStarted;
-    bool    mPSKcInitialized;
+    uint8_t  mExtPanId[kSizeExtPanId];
+    char     mNetworkName[kSizeNetworkName + 1];
+    uint16_t mThreadVersion;
```

Comment:
move this after line 151 so that we can have better alignment.

Reply:
Does alignment matters here? `mThreadVersion` is placed near to `mNetworkName` because they are both properties requested from NCP/RCP device.

I think *place related fields together* is more meaningful than runtime size.
thoughts?